packages/main/src/themes/Label.css

```diff
@@ -1,38 +1,48 @@
 :host(:not([hidden])) {
 	display: inline-flex;
 }
-
 :host {
 	max-width: 100%;
 	color: var(--sapUiContentLabelColor);
 	font-family: var(--sapUiFontFamily);
 	font-size: var(--sapMFontMediumSize);
 	font-weight: normal;
 	cursor: text;
+}	
+:host(:not([wrap])) .ui5-label-root {
+	font-weight: inherit;
+	display: inline-block;
+	white-space: nowrap;
+	cursor: inherit;
+	overflow: hidden;
 }
-
-:host([wrap]) .ui5-label-root {
-	white-space: normal;
-	line-height: 1.4rem;
+:host(:not([wrap])) .label-text-wrapper {
```

Comment:
Sorry I forgot to mention the new css classes does not follow the naming conventions:

Rename those:
.label-text-wrapper -> .ui5-label-text-wrapper
.label-required-colon -> .ui5-label-colon (it`s not only for required label right)